+nosnoc/+discrete_time_problem/Heaviside.m

```diff
@@ -58,6 +66,11 @@ function create_variables(obj)
             %              symbolics to be added in a cell array until a read is done, at which point we call a single vertcat
             %              on the whole queue which is _significantly_ faster.
             % 2d vars: Variables that are defined for each finite element.
+            if opts.use_fesd && opts.use_numerical_clock_state
```

Comment:
what is a numerical clock state? A clock state that is not frozen I guess, but what it is purpose? To have the block diagonal sparsity? Please comment here briefly, and add explanation here: https://github.com/nurkanovic/nosnoc/pull/106/files#diff-8b063ca3da251c42716694289c0a76214cd4575bc0fcb9b2ef58167320fa6c3aR233

Reply:
This is the clock state described in #102. Will add comment.

Reply:
Does the option work reliably? E.g. in simple non-tf PSS problems?

Reply:
Yea it works, though the effect on convergence is mild.